sklearn/mixture/_gaussian_mixture.py

```diff
@@ -728,13 +728,21 @@ def _initialize(self, X, resp):
         elif self.covariance_type == "full":
             self.precisions_cholesky_ = np.array(
                 [
-                    linalg.cholesky(prec_init, lower=True)
+                    np.flipud(
+                        np.fliplr(
+                            linalg.cholesky(np.flipud(np.fliplr(prec_init)), lower=True)
+                        )
+                    )
                     for prec_init in self.precisions_init
                 ]
             )
         elif self.covariance_type == "tied":
-            self.precisions_cholesky_ = linalg.cholesky(
-                self.precisions_init, lower=True
+            self.precisions_cholesky_ = np.flipud(
+                np.fliplr(
+                    linalg.cholesky(
+                        np.flipud(np.fliplr(self.precisions_init)), lower=True
+                    )
+                )
```

Comment:
Same suggestion for this case. You need not duplicate the mathematical derivation in another comment but only indicate to refer to the comment given in the `"full"` case.